pages/reference/hardware/versioning.md

```diff
@@ -12,40 +12,101 @@ In balenaCloud the device type is also used to define what architecture containe
 
 #### Discontinued
-Discontinued device types are those that are no longer actively maintained by the balena team or anyone in the community and will stop showing in the dashboard as options for new fleets and devices. Any already provisioned devices of this type will continue to function, but will no longer receive operating system updates.
+Discontinued device types are those that are no longer actively maintained by the balena team.
+
+This means that:
+
+- No new OS releases will be published for that device type.
+- The device type will stop showing in the dashboard as options for new fleets and devices. 
+- Any already provisioned devices of this type will continue to function, but will no longer receive operating system updates.
+
+While devices will still function when the device type is discontinued, it presents a risk. As no new OS updates will be available, eventually the minumim compatible OS version with balenaCloud will not be available for that device type, after which continued compatibility between the devices and balenaCloud cannot be guaranteed. 
+
+Additionally, discontinued device types present a risk of not having security patches available, and reduced support from the balena support agents. 
+
+##### Why do we discontinue device types
+
+Our aim is to provide the most reliable and secure experience possible across the devices we support. Each device type requires ongoing engineering effort — from continuous integration and testing, to ensuring compatibility with the latest versions of `meta-balena`.
 
-A device type will be discontinued if:
-- The device manufacturer end support and sale of the device.
-- No devices of this type are provisioned on balenaCloud in the last 6 months.
-- If it is a community device type and it has not received maintenance in the last 6 months.
+To maintain a high standard of quality, security, and performance, we periodically review the devices we support. When a device type is no longer actively supported by its hardware vendor or when usage of a device type becomes very limited ( see the section below for more information regarding the criteria for discontinuation), we may decide to discontinue it so that our engineering efforts can have the greatest impact across the devices most actively used by our community.
+
+We make these decisions carefully, and always with the goal of protecting our users from relying on outdated, unsupported, or unavailable hardware.
+
+##### Criteria for discontinuation
+
+A device type will be discontinued based on a set of criteria detailed below, in combination with the judgement of our team:
+- The device manufacturer ends support and sale of the device. This includes discontinuation of sales, as well as BSP support. This would be grounds for immediate discontinuation. 
+- The device type has been on the platform for at least 2 years, with little to no usage. Rougly around 500 devices is the current minimum requirement
```

Comment:
The device type has been on the platform for at least 2 years, with little to no usage. Roughly around 500 devices is the current minimum requirement.